R/version.R

```diff
@@ -468,10 +468,15 @@ version <-
     function()
 {
     tryCatch({
-        packageVersion("BiocVersion")[, 1:2]
-    }, error = function(e) {
-        .version_choose_best()
-    })
+        .get_BiocVersion_version()
+    },
+        warning = function(w) {
+            .version_choose_best()
+        },
+        error = function(e) {
+            .version_choose_best()
+        }
+    )
```

Comment:
I wrote this as
```
    bioc_version <-
        withCallingHandlers{(
            tryCatch({
                .get_BiocVersion_version()
            }, error = function(e) {
                .version_sentinel(conditionMessage(e))
            }), warning = function(w) {
                invokeRestart("muffleWarning")
            }
        })
    if (is.na(bioc_version))
        bioc_version <- .version_choose_best()

    bioc_version
```

Since this withCallingHandlers / tryCatch are duplicated from line 187, perhaps it makes sense to incorporate this in `get_BiocVersion_version()` at line 250?